auth/tests/api/users.rs

```diff
@@ -186,14 +191,12 @@ mod needs_docker {
         assert_eq!(response.status(), StatusCode::OK);
 
         // Test upgrading to pro without a checkout session object.
-        let response = app.put_user("test-user", "pro", "").await;
-        assert_eq!(response.status(), StatusCode::BAD_REQUEST);
+        let response = app.post_subscription("test-user", "", "pro", 1).await;
+        assert_eq!(response.status(), StatusCode::OK);
 
         // Test upgrading to pro with an incomplete checkout session object.
-        let response = app
-            .put_user("test-user", "pro", MOCKED_INCOMPLETE_CHECKOUT_SESSION)
-            .await;
-        assert_eq!(response.status(), StatusCode::BAD_REQUEST);
+        let response = app.post_subscription("test-user", "null", "pro", 1).await;
+        assert_eq!(response.status(), StatusCode::OK);
```

Comment:
Shouldn't these tests that deal with a checkout session be removed?

Reply:
I kept them to ensure the checkout session did not change

Reply:
Hmm, what do you mean by that? We don't send the checkout session to the backend anymore, right? Maybe it's just the comments here confusing me :thinking:

Reply:
Oh, I see what you mean by "checkout session". Let me remove it quick...